docs/user/tutorials/01-40-expose-workload/v2alpha1/01-43-expose-workload-short-host-name.md

```diff
@@ -0,0 +1,127 @@
+# Expose a Workload with Short Host Name
+
+This tutorial shows how to expose an unsecured instance of the HTTPBin Service using short host name without domain and call its endpoints.
+
+> [!WARNING]
+>  Exposing a workload to the outside world is a potential security vulnerability, so tread carefully. In a production environment, always secure the workload you expose with [JWT](../../01-50-expose-and-secure-a-workload/v2alpha1/01-52-expose-and-secure-workload-jwt.md).
+
+## Prerequisites
+
+* [Deploy a sample HTTPBin Service](../../01-00-create-workload.md).
+
+## Steps
+
+### Expose Your Workload
```

Comment:
To keep this section shorter in other tutorials we link to the [existing documentation to setup a custom domain or use the Kyma domain](https://github.com/kyma-project/api-gateway/blob/3e391f21b3f9f35c069c672c93b6007e9fc63c6c/docs/user/tutorials/01-40-expose-workload/v2alpha1/01-40-expose-workload-apigateway.md?plain=1#L11). Is there a reason why don't use it here? Also `{KYMA_DOMAIN_NAME}` is misleading, since it can be any domain.

Reply:
Clarified with Magda, should be consistent with the other tutorial for exposing, choosing between Kyma domain and Custom domain.